samples/text/text.cpp

```diff
@@ -1178,6 +1178,9 @@ MyPanel::MyPanel( wxFrame *frame, int x, int y, int w, int h )
     m_limited->SetMaxLength(8);
     wxSize size2 = m_limited->GetSizeFromTextSize(m_limited->GetTextExtent("WWWWWWWW"));
     m_limited->SetSizeHints(size2, size2);
+    m_limitedMultiline = new MyTextCtrl( this, wxID_ANY, "", wxPoint( 10, 110 ), wxDefaultSize, wxTE_MULTILINE );
```

Comment:
```suggestion
    m_limitedMultiline = new MyTextCtrl(this, wxID_ANY, "", wxPoint(10, 110), wxDefaultSize, wxTE_MULTILINE);
```

No need for the extra spaces in arguments, see [coding guidelines](https://www.wxwidgets.org/develop/coding-guidelines/#spaces_keywords):
`Do not use spaces in these positions for the function calls but still use them to separate function arguments`
 This happens throughout the PR, so I'll only comment it here.